modules/apps/marketplace/marketplace-settings-web/src/main/java/com/liferay/marketplace/settings/web/internal/configuration/MarketplaceConfigurationUtil.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.marketplace.settings.web.internal.configuration;
+
+import com.liferay.portal.configuration.module.configuration.ConfigurationProviderUtil;
+import com.liferay.portal.kernel.configuration.Configuration;
+import com.liferay.portal.kernel.configuration.ConfigurationFactoryUtil;
+
+import java.util.Dictionary;
+
+/**
+ * @author Keven Leone
+ */
+public class MarketplaceConfigurationUtil {
```

Comment:
Is this Class needed? Can't we just call `ConfigurationProviderUtil` passing the `MarketplaceConfiguration.class`?

Reply:
Even better using a Reference to ConfigurationProvider directly if it's in an OSGi component :)